tests/testthat/test-app_feature_01.R

```diff
@@ -54,6 +56,23 @@ describe(
          app$set_inputs("sf_vital_signs-switch_view" = "table")
          output_names <- names(app$get_values(output = TRUE)$output)
          app$expect_values(output = vector_select(output_names, exclude = "visit_figure"))
+         
+         app$run_js("$('#navigate_participants_1-subject_info').click()")
+         app$wait_for_idle(1500)
+         app$set_inputs("navigate_participants_1-participant_selection" = "NLD_06_893")
+         app$wait_for_idle(1000)
+         app$click("navigate_participants_1-subj_apply")
+         app$wait_for_idle()
+         output_names <- names(app$get_values(output = TRUE)$output)
+         app$expect_values(output = vector_select(output_names, exclude = "visit_figure"))
+         
+         ## Note (LSA): after clicking on 'graph' view again at this moment, an 
+         ## error will be shown instead of the expected figures. 
+         ## ('Error: incorrect height and width'). So far, this error could not 
+         ## be isolated and only occurred when running the script above 
+         ## non-interactively. Therefore, no further action is taken.
+         #app$view()
+         #app$set_inputs("sf_vital_signs-switch_view" = "graph")
```

Comment:
@jthompson-arcus can you reproduce this? I cannot reproduce it interactively, only when testing with shinytest2, when placing a browser here and then run `app$set_inputs`. It's not related to this PR (I tested it and it already occurred in an older version as well)

Reply:
Yes, the error message is in the browser only (i.e. it doesn't show up in console). Seems to be connected to setting the width and height of the app driver somehow. Not really sure why it's happening or what to do to fix it and maintain current screen size for tests.